qt/qt_common/map_widget.cpp

```diff
@@ -337,6 +344,68 @@ void MapWidget::initializeGL()
   ASSERT(m_contextFactory == nullptr, ());
   if (!m_screenshotMode)
     m_ratio = devicePixelRatio();
+
+#if defined(OMIM_OS_MAC)
+  m_apiOpenGLES3 = false;
```

Comment:
Looks like this var can be local here, not in MapWidget member scope.

Reply:
`m_apiOpenGLES3` is also used in `CreateEngine()` and any locally set value should somehow get out of the scope of `initializeGL()` which cannot change signature, because it is an [implementation of a virtual parent function](https://doc.qt.io/qt-5/qglwidget.html#initializeGL).
So if we can't take in output parameter, and can't return anything then, I don't see any better option than a member field.

Reply:
My bad, you are right.

Reply:
OK, then I have flipped `GLES3` on for Mac.